server/src/utils/getConfig.ts

```diff
@@ -56,13 +98,25 @@ const getPostCallConfig = (defaultConfig: ServerConfigModel): PostCallConfig | u
 
   try {
     const postcallSurveyType = process.env[VV_POSTCALL_SURVEY_TYPE_ENV_NAME] ?? defaultConfig.postCall?.survey.type;
-    const postcallSurveyUrl =
-      process.env[VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME] ?? defaultConfig.postCall?.survey.options?.surveyUrl;
-
-    if (postcallSurveyType && isValidPostCallSurveyType(postcallSurveyType) && postcallSurveyUrl) {
-      postcallConfig = {
-        survey: { type: postcallSurveyType, options: { surveyUrl: postcallSurveyUrl } }
-      };
+    if (postcallSurveyType && isValidPostCallSurveyType(postcallSurveyType)) {
+      if (postcallSurveyType === 'msforms') {
+        const configOptions: MSFormsSurveyOptions = getMSFormsOptions(defaultConfig);
+        postcallConfig = {
+          survey: { type: postcallSurveyType, options: configOptions }
+        };
+      } else if (postcallSurveyType === 'custom') {
+        const configOptions: CustomSurveyOptions = getCustomSurveyOptions(defaultConfig);
+        postcallConfig = {
+          survey: { type: postcallSurveyType, options: configOptions }
+        };
+      } else if (postcallSurveyType === 'onequestionpoll') {
```

Comment:
I think we are missing some tests cases.

From what I can tell we need at least two test cases per postCallSurveyType:
- one that uses env vars
- one that uses defaultConfig

In other words, we need to test the logic inside of `getMSFormsOptions`, `getCustomSurveyOptions`, and `getOneQuestionPollOptions`.